labs/SettingsControls/src/SettingsExpander/SettingsExpander.xaml

```diff
@@ -89,7 +89,10 @@
                                                    IsClickEnabled="False" />
                             </muxc:Expander.Header>
                             <muxc:Expander.Content>
-                                <ItemsPresenter TabFocusNavigation="Local" />
+                                <StackPanel>
+                                    <ItemsPresenter TabFocusNavigation="Local" />
```

Comment:
Wonder if [ItemsPresenter.Footer](https://learn.microsoft.com/uwp/api/windows.ui.xaml.controls.itemspresenter.footer) would be better here? (or at least simplify)

Reply:
Discussing more let's call these properties `ItemsFooter` and add an `ItemsHeader` and then bind them to the ItemsPresenter (assuming that works). Gives us the most flexibility for future expansion.

We can later use `Footer` to be an attached item to the Expander outside of the expander itself.

Reply:
Done!